src/components/profile/ConnectedAccounts.tsx

```diff
@@ -153,6 +153,8 @@ const DiscordConnection: React.FC = () => {
           )}&response_type=code&scope=identify%20guilds&state=${Buffer.from(
             JSON.stringify({ origin: OnboardingSubject.DiscordConnectedInPage })
           ).toString('base64')}`}
+          target="_blank"
```

Comment:
this should open on the same page as the user would get redirected back when they complete signing in to discord.

Reply:
I.e take out the target : _blank